modules/extensions/process/publisher/process/apis/upload_bpmnEditorDiagram.jag

```diff
@@ -0,0 +1,30 @@
+<%
+/*
+ *  Copyright (c) 2015, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
As per the commit, it seems above file is created in 2016. Then any reason for specifying as 2015 here ??